examples/05 Introduction to Libraries.ipynb

```diff
@@ -0,0 +1,382 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>E-Commerce</td>\n",
+       "      <td>offer products; assign order</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Truck CompanyTest</td>\n",
+       "      <td>receive order; deliver product</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Client</td>\n",
+       "      <td>select product; get product</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           component                       functions\n",
+       "0     Logical System           no functions assigned\n",
+       "1         E-Commerce    offer products; assign order\n",
+       "2  Truck CompanyTest  receive order; deliver product\n",
+       "3             Client     select product; get product"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "  \n",
+    "data = []\n",
+    "for component in model.la.all_components:\n",
+    "    component_functions = \"; \".join([function.name for function in component.functions] or [\"no functions assigned\"])\n",
+    "    data.append(dict(component=component.name, functions=component_functions))\n",
+    "       \n",
+    "df = pd.DataFrame(data)\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exchange items"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exchange items allow to detail functions exchanges. So, let's first list all functions exchanges."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<FunctionalExchange 'order details' (a394e783-5c1b-4071-b6b2-caaaffd8bfe6)>\n",
+      "<FunctionalExchange 'supply details' (8f59a1e7-2b1c-47a7-9d9b-39516defd8c5)>\n",
+      "<FunctionalExchange 'product list' (ba2c6da9-ee3d-45e5-9fb2-38bcb61b8443)>\n"
+     ]
+    }
+   ],
+   "source": [
+    "for function_exchange in model.la.all_function_exchanges:\n",
+    "    print(function_exchange)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exemplarily, we added them to the function exchange, supply details, in the LAB diagram."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "* <ExchangeItem 'ordered products' (b852ca69-cb56-40a2-a1ab-91c351ddabfe)>\n"
+     ]
+    }
+   ],
+   "source": [
+    "supply_details = model.la.all_function_exchanges[1]\n",
+    "print(supply_details.exchange_items)"
+   ]
+  },
```

Comment:
These are standard model functions and have nothing to with libraries (AFAICT). If so, we should trim these cells away.